packages/shared/src/components/auth/AuthContainer.tsx

```diff
@@ -2,10 +2,7 @@ import classNames from 'classnames';
 import React, { forwardRef, HTMLAttributes, MutableRefObject } from 'react';
 import classed from '../../lib/classed';
 
-const Container = classed(
-  'div',
-  'grid grid-cols-1 gap-3 self-center mt-6 w-full',
-);
+const Container = classed('div', 'flex flex-col gap-3 self-center mt-6 w-full');
```

Comment:
This one is used quite often right, just something to be aware of :)

Reply:
Yep. The goal was to have a column, hence, flex-col should be alright.